### PR TITLE
Match commas in the URLs for srcset

### LINKED ISSUE #469: Allow commas in srcset URLs

### PR DESCRIPTION
**Pull request by dvoytenko**
Closes #469.

### CODE REVIEW

File: test/functional/test-srcset.js [hunk 1]
```diff
@@ -78,6 +78,54 @@ describe('Srcset parseSrcset', () => {
     expect(res.sources_[2].dpr).to.equal(undefined);
   });
 
+  it('should accept commas in URLs', () => {
+    var res = parseSrcset(' \n image,1 100w\n , \n image,2 50w \n');
```
Comment (cramforce) [inline test/functional/test-srcset.js]: Add more examples here with some kind of test helper function to make that only a few lines.

File: src/srcset.js [hunk 2]
```diff
@@ -37,11 +37,15 @@ var SrcsetSource;
  * @return {!Srcset}
  */
 export function parseSrcset(s) {
-  let sSources = s.split(',');
+  let sSources = s.match(/\s*([^\s]*)(\s+(\d+(\.(\d+)?)?[wx]))?(\s*,)?/g);
```
Comment (cramforce) [inline src/srcset.js]: Add comment!

Comment (cramforce) [inline src/srcset.js]: What this is parsing. Some examples.

Comment (dvoytenko) [inline src/srcset.js]: Done

**Comment by dvoytenko** [comment]
Added a lot more tests.

File: src/srcset.js [hunk 3]
```diff
@@ -37,11 +37,19 @@ var SrcsetSource;
  * @return {!Srcset}
  */
 export function parseSrcset(s) {
-  let sSources = s.split(',');
+  // General grammar: (URL [NUM[w|x]],)*
+  // Example 1: "image1.png 100w, image2.png 50w"
```
Comment (cramforce) [inline src/srcset.js]: Add coma into filename as special case.

**Comment by cramforce** [comment]
LGTM. One more comment.

File: src/srcset.js (hunk 3 above)
Comment (dvoytenko) [inline src/srcset.js]: Done

**Pull request merged**